lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+
+ MSG_AUTOLOADING_ENABLED c=20 r=4
+#: ultralcd.cpp:2648
+msgid "Autoloading filament is active, just press the knob and insert filament..."
+msgstr "Autoladdning filament \xe1r aktiv, tryck pa knappen och s\xe1tt i filament..."
+
+# MSG_SELFTEST_AXIS_LENGTH c=20
+#: ultralcd.cpp:8173
+msgid "Axis length"
+msgstr "Axell\xe1ngd"
+
+# MSG_SELFTEST_AXIS c=16
+#: ultralcd.cpp:8174
+msgid "Axis"
+msgstr "Axel"
+
+# MSG_SELFTEST_BEDHEATER c=20
+#: ultralcd.cpp:8131
+msgid "Bed/Heater"
+msgstr "B\xe1dd/V\xe1rmare"
+
+# MSG_BED_DONE c=20
+#: messages.c:15
+msgid "Bed done"
+msgstr "B\xe1dd klar"
+
+# MSG_BED_HEATING c=20
+#: messages.c:16
+msgid "Bed Heating"
+msgstr "B\xe1dden v\xe1rms upp"
+
+# MSG_BED_CORRECTION_MENU c=18
+#: ultralcd.cpp:5798
+msgid "Bed level correct"
+msgstr "B\xe1ddnivakorrigering"
+
+# MSG_BELTTEST c=18
+#: ultralcd.cpp:5778
+msgid "Belt test"
+msgstr "B\xe1lt test"
+
+# MSG_BED_LEVELING_FAILED_POINT_LOW c=20 r=6
+#: messages.c:17
+msgid "Bed leveling failed. Sensor didn't trigger. Debris on nozzle? Waiting for reset."
+msgstr "B\xe1ddnivelleringen felade. Sensorn utl\xefstes inte. Skr\xe1p pa munstycket? V\xe1ntar pa aterst\xe1llning."
+
+# MSG_BRIGHT c=6
+#: messages.c:158
+msgid "Bright"
+msgstr "Ljus"
+
+# MSG_BRIGHTNESS c=18
+#: messages.c:154
+msgid "Brightness"
+msgstr "Ljusstyrka"
+
+# MSG_BED c=13
+#: messages.c:14
+msgid "Bed"
+msgstr "B\xe1dd"
+
+# MSG_BELT_STATUS c=18
+#: messages.c:19
+msgid "Belt status"
+msgstr "B\xe1lt status"
+
+# MSG_RECOVER_PRINT c=20 r=2
+#: messages.c:82
+msgid "Blackout occurred. Recover print?"
+msgstr "Blackout intr\xe1ffat. Aterst\xe1lla utskr?"
+
+# MSG_CALIBRATING_HOME c=20
+#: ultralcd.cpp:8492
+msgid "Calibrating home"
+msgstr "Kalibrerar hem"
+
+# MSG_CALIBRATE_BED c=18
+#: ultralcd.cpp:5787
+msgid "Calibrate XYZ"
+msgstr "Kalibrerar XYZ"
+
+# MSG_HOMEYZ c=18
+#: messages.c:52
+msgid "Calibrate Z"
+msgstr "Kalibrera Z"
+
+# MSG_CALIBRATE_PINDA c=17
+#: ultralcd.cpp:4529
+msgid "Calibrate"
+msgstr "Kalibrerar"
+
+# MSG_CANCEL2 c=10
+#: messages.c:20
+msgid ">Cancel"
+msgstr ">Avbryt"
+
+# MSG_MOVE_CARRIAGE_TO_THE_TOP c=20 r=8
+#: ultralcd.cpp:3318
+msgid "Calibrating XYZ. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
+msgstr "Kalibrerar XYZ. Vrid vredet f\xefr att flytta Z-vagnarna upp till \xe1ndstoppen. Klicka n\xe1r du \xe1r klar."
+
+# MSG_CALIBRATE_Z_AUTO c=20 r=2
+#: messages.c:21
+msgid "Calibrating Z"
+msgstr "Kalibrerar Z"
+
+# MSG_HOMEYZ_DONE c=20
+#: ultralcd.cpp:656
+msgid "Calibration done"
+msgstr "Kalibraring utf\xefrd"
+
+# MSG_MENU_CALIBRATION c=18
+#: messages.c:68
+msgid "Calibration"
+msgstr "Kalibrering"
+
+# MSG_SD_REMOVED c=20
+#: ultralcd.cpp:8939
+msgid "Card removed"
+msgstr "Kort borttaget"
+
+# MSG_CHECKING_FILE c=17
+#: ultralcd.cpp:8580
+msgid "Checking file"
+msgstr "Kontrollerar fil"
+
+# MSG_NOT_COLOR c=19
+#: ultralcd.cpp:2565
+msgid "Color not correct"
+msgstr "F\xe1rg ej korrekt"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.